app.js

```diff
@@ -2,3 +2,105 @@
 //  to serve up static files from the public directory (index.html and client.js - you 
 //  can also serve up css files from the public directory this way if you want)
 //  you need to support a '/trucks' endpoint, and a dynamic route for '/trucks/:name'
+
+var express = require('express');
+var trucks = require('./trucks');
+var _ = require('underscore');
+var app = express();
+ 
+app.use(express.static('public'));
+
+app.get('/', function (request, response) {
+    response.redirect('/trucks');
```

Comment:
This is unnecessary. The use of express.static('public') as middle-ware means the root route will already be processed by the time this route handler would fire.